rmw_connext_cpp/package.xml

```diff
@@ -13,14 +13,15 @@
   <buildtool_export_depend>rosidl_cmake</buildtool_export_depend>
 
   <build_depend>connext_cmake_module</build_depend>
-  <build_depend>libndds51-dev</build_depend>
+  <build_depend>libndds52-dev</build_depend>
   <build_depend>rmw</build_depend>
+  <build_depend>rmw_connext_shared_cpp</build_depend>
```

Comment:
I added this package as an `exec_depend` in 5eb0a187bd865936049dfead1bc6ee4fe22d0392